lib/widgets/scoll_chainer.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/material.dart';
+import 'package:logging/logging.dart';
+
+final _logger = Logger('ScrollChainer');
+
+// Need to keep track of whether the user is scrolling or not, otherwise
+// the scroll animation will be overridden by the user's scroll.
+// Not using a stateful widget because it would cause an unnecessary rebuild
+// of the widget tree.
+// ignore: must_be_immutable
+class ScrollChainer extends StatelessWidget {
```

Comment:
This is _definitely_ already in master.
Maybe rebase on latest master and see if everything still looks good before merging?

Reply:
Oh, I also just noticed a slight spelling error in the file name. Looks like scroll is missing an r